File: C++ Code/Basics/Calculator/Compound Interest.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main(){
    float p,t,r;
    cout<<"Enter the principle amount : ";
    cin>>p;
    cout<<"Enter the year : ";
    cin>>t;
    cout<<"Enter the rate : ";
    cin>>r;
    float TA=p*pow(1+r/100,t);
    cout<<"Compound Interest after "<<t<<" years = "<<TA-p;
    cout<<endl<<"Total amount = "<<TA;
    return 0;
}

File: Java Code/Basics/Calculator/CompoundInterest.java
import java.util.*;

public class CompoundInterest{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter the principal amount : ");
    float p=sc.nextFloat();
    System.out.print("Enter anual rate percent : ");
    float r=sc.nextFloat();
    System.out.print("Enter the time period : ");
    float t=sc.nextFloat();
    float TA=(float)(p*Math.pow(1+r/100,t));
    System.out.println("Compound Interest : "+(TA-p));
    System.out.println("Total amount : "+(TA));
  }
}